src/main/resources/application.properties

```diff
@@ -39,12 +39,15 @@ spring.datasource.driver-class-name=herddb.jdbc.Driver
 # HerdDB - local in memory-only
 #spring.datasource.url=jdbc:herddb:local
 # HerdDB - start embedded server, data persisted on local disk (directory 'dbdata'), listening on localhost:7000
-spring.datasource.url=jdbc:herddb:server:localhost:7000?server.start=true&server.base.dir=dbdata
+#spring.datasource.url=jdbc:herddb:server:localhost:7000?server.start=true&server.base.dir=dbdata
+# HerdDB - start embedded server 'diskless-cluster' mode, WAL and Data persisted on Bookies, Metadata on ZooKeeper in '/herd', listening on localhost:7000
+#spring.datasource.url=jdbc:herddb:zookeeper:localhost:2181?server.start=true&server.base.dir=dbdata&server.mode=diskless-cluster&server.node.id=localhost
```

Comment:
Is it normal that in the new config we don't have any "spring.datasource.url" config uncommented?

Reply:
good catch !
I have restored the commented line

thanks @aluccaroni